File: java/integracion_con_base_de_datos/ordenar_contactos_por_nombre_o_categoria/src/ordenar_contactos_por_nombre_o_categoria/conexion.java
package ordenar_contactos_por_nombre_o_categoria;
import java.sql.Connection;
import java.sql.DriverManager;
public class conexion {
    private static final String URL = "jdbc:mysql://localhost:3306/ordenar_los_contactos_por_nombre_o_categoria";
    private static final String USER = "root";
    private static final String PASSWORD = "";
    public static Connection conectar() {
        Connection conn = null;
        try {
            Class.forName("com.mysql.cj.jdbc.Driver");
            conn = DriverManager.getConnection(URL, USER, PASSWORD);
            System.out.println("✅ Conexión exitosa a la base de datos");
        } catch (Exception e) {
            System.out.println("❌ Error de conexión: " + e.getMessage());
        }
        return conn;
    }
}

File: python/mostrar_contactos/conexion.py
import mysql.connector
def conectar():
    try:
        conexion = mysql.connector.connect(
            host="localhost",
            user="root",
            password="",
            database="mostrar_contactos"
        )
        return conexion
    except mysql.connector.Error as e:
        print("Error al conectar:", e)
        return None